CHANGELOG.md

```diff
@@ -270,6 +270,9 @@ All notable changes to this project will be documented in this file.
 - Add overridable Axis.FormatValueOverride (#181)
 - PngExporter text formatting (#170)
 
+### Added
+  Added additional line legend positions to line series. It was previously only Start and End.
```

Comment:
Should probably be in the `Unreleased` section at the top.

Reply:
I moved this to the unreleased section.